src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.cpp

```diff
@@ -690,73 +730,102 @@ bool TypeLookupManager::receive_request(
     change.serializedPayload.encapsulation = static_cast<uint16_t>(encapsulation);
     msg.pos += 2; // Skip encapsulation options.
 
-    SerializedPayload_t payload;
+    //SerializedPayload_t payload;
```

Comment:
Please, remove unnecessary code.

Reply:
removed in [d054d39](https://github.com/eProsima/Fast-DDS/pull/4240/commits/d054d39484489f3890638c8cfe4fe4c023139c74)